wiki/osu!tourney/Setup/id.md

```diff
@@ -0,0 +1,39 @@
+# Pemasangan
+
+_Halaman Utama: [osu!tourney](/wiki/osu!tourney)_
+
+**Catatan:** Saat ini supporter tag yang aktif diperlukan untuk menjalankan klien osu!tourney.
+
+Penggunaan **klien osu! yang masih baru** sangat disarankan. Hal ini dikarenakan database Lagu mungkin mengalami kerusakan. Anda dapat memiliki beberapa klien osu!, tapi simpan di folder terpisah. 
```

Comment:
I don't really think that the "L" in "Lagu" should be capitalized, since in the Indonesian localization the "osu!/Songs" folder doesn't really get translated into "osu!/Lagu"

A better option would probably be "database lagu Anda (osu!/Songs) mungkin mengalami kerusakan" but yeah the Lagu shouldn't be capitalized here